approvals/approvals/utils.py

```diff
@@ -0,0 +1,25 @@
+import frappe
+
+
+def add_approval_comment(
```

Comment:
@agritheory I created a new comment API since none of the existing ones let you set a `subject` value, which would get weird fast if you had to set it in multiple places.

Reply:
@Alchez Custom field I think. There can be inference when there's NOT a custom workflow - "Draft" / docstatus == 0.